src/linker/runner.py

```diff
@@ -38,6 +39,9 @@ def main(
         ## See above
         "--envvars",
         "foo",
+        ## Suppress some of the snakemake output
+        "--quiet",
+        "progress",
```

Comment:
Is this saying to suppross some output (`--quiet`) but to show progress...bars?

Reply:
"progress" is the argument to "--quiet", it's suppressing some logging about the progress of execution